app/src/main/res/layout/plain_activity.xml

```diff
@@ -103,13 +79,28 @@
                     android:visibility="gone"
                     tools:ignore="HardcodedText" />
 
+                <ProgressBar
+                    android:id="@+id/progress_horizontal"
+                    style="@style/Widget.AppCompat.ProgressBar.Horizontal"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:indeterminate="true"
+                    android:visibility="gone" />
+
+                <TextView
+                    android:layout_width="match_parent"
```

Comment:
why do we have a text view without an id? How would we even access it to add text?

Reply:
Removed. Probably a remnant of a previous sample.